addons/live-comments/script-project-studios.js

```diff
@@ -0,0 +1,158 @@
+const wait = (ms) => new Promise((resolve) => setTimeout(resolve, ms));
+
+export default async function ({ addon, global, console, msg }) {
+  let eButton = document.createElement("button");
+  let enabled = false;
+
+  async function addButton() {
+    await addon.tab.waitForElement("h4>span, h2>span", { markAsSeen: true });
+    let commentsText = Array.from(document.querySelectorAll("h4>span, h2>span")).find(
+      (el) => el.innerText == "Comments"
```

Comment:
This would not work with other languages, this should be a translation string

Reply:
These should also be used for other text here, but this one would actually break with other languages. Same thing applies for `script.js`.